install.sh

```diff
@@ -10,6 +10,8 @@
 # Script options:
 # --build-icu              Builds ICU libraries.
 # --build-boost            Builds Boost libraries.
+# --build-qrencode         Builds QREncode libraries.
+# --build-libpng           Builds LIBPNG libraries.
```

Comment:
Changes to the installer need to be applied to [libbitcoin-build/templates/gsl.install.sh](https://github.com/libbitcoin/libbitcoin-build/blob/master/templates/gsl.install.sh).

These comments for example are conditionally-applied in [this manner](https://github.com/libbitcoin/libbitcoin-build/blob/master/templates/gsl.install.sh#L110-L115).

I can help with this if need be.